frontend/src/metabase/visualizations/components/legend/LegendLabel.tsx

```diff
@@ -27,6 +34,13 @@ export const LegendLabel = ({ children, className, href, onClick }: Props) => {
     },
     [onClick],
   );
+  const [href, setHref] = useState<LinkProps["to"]>();
+
+  useEffectOnce(() => {
+    if (getHref) {
+      setHref(getHref());
+    }
+  });
```

Comment:
:x: `useEffectOnce` won't work as hrefs depend on dashboard parameters which can change during the lifecycle of the component.

In the video it is expected that last link will take you to a products table filtered by ID = 5:


https://github.com/metabase/metabase/assets/6830683/ae968ffe-6be6-4fc8-be60-5111d95cdbec